src/accounts/context/userAccount.tsx

```diff
@@ -31,10 +31,18 @@ import {updateDefaultQuartzAccount} from 'src/identity/apis/auth'
 export type Props = {
   children: JSX.Element
 }
+
+interface SetDefaultAccountOptions {
```

Comment:
Users currently switch accounts from the switch-account page. We are re-using the switch-account context provider in order to change the default account in User Profile. What we _don't_ want are the wordy notifications generated by the context provider - we want to use the concise ones generated by UI on the User Profile page.

Fix I propose here is: add an `options` object to the handler for setting a default account, that lets us disable the dispatching of notifications from the context provider. When that handler is called from User Profile, this disables the dispatching, so that we don't get the context provider's notifications.

Other ways to fix this behavior could be:
1. Delete the notifications from the switch-account context-provider, and have them be dispatched from the UI in the switch-account page. That's better - the provider probably shouldn't be dispatching the notifications - but there's no need to introduce new code for that when the switch-account page is about to get replaced by user profile.

3. Change the notifications being dispatched from the context provider so that they yield the same message that's expected on the user profile page. I wouldn't do that, both because it changes what appears on the Switch-Account page from what the ACs for that page outline, and because we want a _single_ notification on the User Profile page regardless of whether the user changes their default account, default org-within-an-account, or both.